### ISSUE TITLE
Timestamps under a node when activated seem to be stuck in GMT

### ISSUE THREAD

**Issue by simonk83** [zachowj/node-red-contrib-home-assistant-websocket#112]
**Describe the bug**
I'm running the official docker in a VM, and have `- /etc/localtime:/etc/localtime:ro` set, but also added `environment: - TZ=Australia/Melbourne` just to try and rule that out (didn't help). I also run HassIO on another VM with the same parameters set. On both VM's, checking the date in the terminal returns the correct time and timezone, and exec'ing into the docker containers and checking the date is also correct, so it doesn't appear to be an issue at my end (though it still could be something else I suppose).

Debug timestamps in Node-RED are correct, it only seems to happen, as far as I can tell, when interacting with Homeassistant nodes.

**To Reproduce**

1. Go to a non GMT timezone :)
2. Add a current state node and link to HA
3. Trigger that node
4. Timestamp under the node will be GMT rather than local time

**Expected behavior**
Timestamp under the node should be in local time

**Screenshots**
![image](https://user-images.githubusercontent.com/518440/56454909-5a71d100-639b-11e9-81b2-5f23ece306bd.png)

Those times are all GMT. I'm in AEST.

**Environment (please complete the following information):**

- Node Red Version: 0.20.5
- NR Home Assistant Plugin Version: 0.10.2
- Is Node Red running in Docker: yes

**Other (please complete the following information):**

- Have you searched previous issues for duplicates?: Yes

**Comment by zachowj** [comment]
This is working correctly in my three tests which all show the correct status time depending on which timezone is set.

- Not setting timezone
- Setting `TZ=America/New_York`
- Settings `TZ=Australia/Melbourne`

You can test this yourself by cloning the repo and editing the included docker/docker-compose.yaml file and running `npm run docker:up`
```
version: "3"

services:
  node-red:
    build: ./node-red
    command: npm run dev:watch
    environment:
      # - TZ=America/New_York
      - TZ=Australia/Melbourne
    volumes:
      - "..:/data/node_modules/node-red-contrib-home-assistant-websocket"
    #   - './node-red/root-fs/data/flows.json:/data/flows.json'   # Map flows file for easy commitable examples building
    ports:
      - 1880:1880
      - 9123:9229

  home-assistant:
    build: ./home-assistant
    # While messing with HA config map the repo config to the container for easy changes
    volumes:
      - "./home-assistant/root-fs/config/configuration.yaml:/config/configuration.yaml"
    ports:
      - 8300:8300
      - 8123:8123
```

**Comment by simonk83** [comment]
Hi there,

Thanks, I was able to fix it afterall.  I changed two things in my compose file, 
1 - I got rid of `- /etc/localtime:/etc/localtime:ro` and just left the TZ env variable, and 
2 - Removed `user: root` and the volume `- /root/.ssh:/root/.ssh:ro` (I borrowed someone elses config initially, not too sure why he had these in there anyway)

So one, or both of those fixed it :)  Thanks for your help!

**Comment by flamingm0e** [comment]
Sounds like you used my config. 

I'll explain. 

`user: root` was there to get around file permissions on the directory I mounted `/data` in. I run my system as root (Alpine), which brings me to the ssh volume I mounted. 

I mounted ssh so that I could use my ssh keys for GitHub integration. 

I have been running with `/etc/localtime` for 2 years without issue, but updated recently and started having issues with time being off by 2-7 hours. 

It was definitely the `TZ` environment variable that fixed it.
